opentech/public/funds/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class FundsConfig(AppConfig):
+    name = 'opentech.public.funds'
+    label = 'public_funds'
```

Comment:
the templates need to reflect this. Currently `TemplateDoesNotExist public_funds/fund_page.html`